src/prommis/solvent_extraction/sx_stripping_flowsheet_steady.py

```diff
@@ -0,0 +1,107 @@
+from pyomo.environ import ConcreteModel, SolverFactory
```

Comment:
Rather than separate sub-flowsheets for each section of the SX system, could you create a single flowsheet that represents all parts of a single SX unit operation (loading, cleaning and stripping). This will be important both as a general example for users on how to model a full SX operation (e.g. it is easy for a user to take one of these flowsheets and assume it represents a full SX system when in fact it is only a part of the system), as well as for you to learn how to set up recycle streams.

Reply:
I already have a version of this flowsheet in #47. It's just called `sx_model_flowsheet.py`. I created it while setting up the new UKy flowsheet, so I'm not sure if it'd be a good use of Arko's time to re-create this...

Reply:
I think it is an important step for Arko in order to learn how to do recycle streams himself. As you probably realize from our discussions, there are a lot of catches to be aware of and the best way to learn is to try doing it yourself.